test/module/irohad/ordering/ordering_gate_test.cpp

```diff
@@ -73,32 +73,22 @@ class OrderingGateTest : public ::testing::Test {
       auto address = "0.0.0.0:" + std::to_string(port);
       // Initialize components after port has been bind
       transport = std::make_shared<OrderingGateTransportGrpc>(address);
-      gate_impl = std::make_shared<OrderingGateImpl>(transport);
+      gate_impl = std::make_shared<OrderingGateImpl>(transport, 1, false);
```

Comment:
Maybe define `1` and `false` as some constants here, because they are used multiple times in this file?